rust-sdk/utils/src/types/tick.rs

```diff
@@ -0,0 +1,7 @@
+pub type TickIndex = i32;
+pub type TickSpacing = u16;
+
+pub struct TickRange {
```

Comment:
would be good to get some basic derives on this like `Copy, Clone, Debug, PartialEq, Eq`